src/partials/about.html

```diff
@@ -1,21 +1,38 @@
-<section id="how its made" class="container">
-  <p class="about">TRADICIONAL AND LOVE</p>
+<div class="bg">
+  <section id="how its made">
+    <div class="container about--bgcolor">
+      <span class="about__title">TRADICIONAL AND LOVE</span>
+      <h2 class="about">HOW IT`S MADE ?</h2>
 
-  <p class="about__title1">HOW IT`S MADE ?</p>
-  <div class="about__title">
-    <img class="about__img" src="./images/home-pic4.png" width="" height="" alt="" />
 
-    <div class="about__text-wrapper">
-      <p class="about__text">
-        Fusce ut velit laoreet, tempus arcu eu, molestie tortor. Nam vel justo cursus, faucibus
-        lorem eget, egestas eros. Maecenas eleifend erat at justo fringilla.
-      </p>
-      <p class="about__text about__text--s">
-        Curabitur lacinia enim at ex blandit, vel pellentesque odio elementum. Mauris rhoncus orci
-        in imperdiet placerat. Vestibulum euismod nisl suscipit ligula volutpat, a feugiat urna
-        maximus. Cras massa nibh, tincidunt. Aliquam erat volutpat. Aenean accumsan.
-      </p>
-      <button class="btn" type="button">Read more</button>
+      <div class="about__wrapper">
```

Comment:
musisz wyjąć button z wrappera, bo w wersji tablet przycisk jest pod tekstem, albo możesz to inaczej ugryźć, pokombinuj